lib/webmock/http_lib_adapters/net_http.rb

```diff
@@ -37,6 +43,16 @@ def self.disable!
         end
       end
 
+      def self.remove_silently(mod, const) #:nodoc:
+        begin
+          # Don't warn on removing the deprecated constant
+          verbose, $VERBOSE = $VERBOSE, nil
+          mod.send(:remove_const, const)
+        ensure
+          $VERBOSE = verbose
+        end
+      end
```

Comment:
My two cents, WebMock supports Ruby 2.5+, the redundant `begin` should no longer be necessary.
https://github.com/bblimke/webmock/blob/v3.24.0/webmock.gemspec#L24

```suggestion
      def self.remove_silently(mod, const) #:nodoc:
        # Don't warn on removing the deprecated constant.
        verbose, $VERBOSE = $VERBOSE, nil
        mod.send(:remove_const, const)
      ensure
        $VERBOSE = verbose
      end
```